src/service/bill_service.rs

```diff
@@ -236,15 +233,13 @@ impl BillService {
         identity_store: Arc<dyn IdentityStoreApi>,
         file_upload_store: Arc<dyn FileUploadStoreApi>,
         bitcoin_client: Arc<dyn BitcoinClientApi>,
-        identity_blockchain_store: Arc<dyn IdentityChainStoreApi>,
     ) -> Self {
         Self {
             client,
             store,
             identity_store,
             file_upload_store,
             bitcoin_client,
-            identity_blockchain_store,
```

Comment:
Also a problem while merging - you removed the `identity_blockchain_store` with the merge, but it's needed.

Reply:
you mean `identity_store`

Reply:
No, `identity_store` is still there in your code, but your removed `identity_blockchain_store`. This is a different store, both are needed. (You can check the code in the master branch for reference.)